script/DeployCometWrapper.s.sol

```diff
@@ -1,64 +1,189 @@
 
 // Deploy with:
 // $ set -a && source .env && ./script/deploy.sh
 
 // Required ENV vars:
 // RPC_URL
 // DEPLOYER_PK
-// COMET_ADDRESS
-// REWARDS_ADDRESS
-// PROXY_ADMIN_ADDRESS
-// TOKEN_NAME
-// TOKEN_SYMBOL
 
 // Optional but suggested ENV vars:
 // ETHERSCAN_KEY
 
 contract DeployCometWrapper is Script {
-    ProxyAdmin proxyAdmin;
+    address[] COMET_ADDRESS_MAINNET=[0x3Afdc9BCA9213A35503b077a6072F3D0d5AB0840, 0x3D0bb1ccaB520A66e607822fC55BC921738fAFE3, 0xA17581A9E3356d9A858b789D68B4d866e593aE94, 0xc3d688B66703497DAA19211EEdff47f25384cdc3, 0x5D409e56D886231aDAf00c8775665AD0f9897b56];
+    string[] TOKEN_NAME_MAINNET=["Wrapped Comet USDT", "Wrapped Comet wstETH", "Wrapped Comet WETH", "Wrapped Comet USDC", "Wrapped Comet USDS"];
+    string[] TOKEN_SYMBOL_MAINNET=["wcUSDTv3", "wcWstETHv3", "wcWETHv3", "wcUSDCv3", "wcUSDSv3"];
+    address REWARDS_ADDRESS_MAINNET=0x1B0e765F6224C21223AeA2af16c1C46E38885a40;
+    address PROXY_ADMIN_ADDRESS_MAINNET=0x1EC63B5883C3481134FD50D5DAebc83Ecd2E8779;
+
+    address[] COMET_ADDRESS_OPTIMISM=[0x2e44e174f7D53F0212823acC11C01A11d58c5bCB, 0x995E394b8B2437aC8Ce61Ee0bC610D617962B214, 0xE36A30D249f7761327fd973001A32010b521b6Fd];
+    string[] TOKEN_NAME_OPTIMISM=["Wrapped Comet USDC", "Wrapped Comet USDT", "Wrapped Comet WETH"];
+    string[] TOKEN_SYMBOL_OPTIMISM=["wcUSDCv3", "wcUSDTv3", "wcWETHv3"];
+    address REWARDS_ADDRESS_OPTIMISM=0x443EA0340cb75a160F31A440722dec7b5bc3C2E9;
+    address PROXY_ADMIN_ADDRESS_OPTIMISM=0x3C30B5a5A04656565686f800481580Ac4E7ed178;
+
+    address[] COMET_ADDRESS_POLYGON=[0xF25212E676D1F7F89Cd72fFEe66158f541246445, 0xaeB318360f27748Acb200CE616E389A6C9409a07];
+    string[] TOKEN_NAME_POLYGON=["Wrapped Comet USDC", "Wrapped Comet USDT"];
+    string[] TOKEN_SYMBOL_POLYGON=["wcUSDCv3", "wcUSDTv3"];
+    address REWARDS_ADDRESS_POLYGON=0x45939657d1CA34A8FA39A924B71D28Fe8431e581;
+    address PROXY_ADMIN_ADDRESS_POLYGON=0xd712ACe4ca490D4F3E92992Ecf3DE12251b975F9;
+
+    address[] COMET_ADDRESS_BASE=[0xb125E6687d4313864e53df431d5425969c15Eb2F, 0x9c4ec768c28520B50860ea7a15bd7213a9fF58bf, 0x46e6b214b524310239732D51387075E0e70970bf, 0x784efeB622244d2348d4F2522f8860B96fbEcE89];
+    string[] TOKEN_NAME_BASE=["Wrapped Comet USDC", "Wrapped Comet USDbC", "Wrapped Comet WETH", "Wrapped Comet AERO"];
+    string[] TOKEN_SYMBOL_BASE=["wcUSDCv3", "wcUSDbCv3", "wcWETHv3", "wcAEROv3"];
+    address REWARDS_ADDRESS_BASE=0x123964802e6ABabBE1Bc9547D72Ef1B69B00A6b1;
+    address PROXY_ADMIN_ADDRESS_BASE=0xbdE8F31D2DdDA895264e27DD990faB3DC87b372d;
+
+    address[] COMET_ADDRESS_ARBITRUM=[0xA5EDBDD9646f8dFF606d7448e414884C7d905dCA, 0x9c4ec768c28520B50860ea7a15bd7213a9fF58bf, 0x6f7D514bbD4aFf3BcD1140B7344b32f063dEe486, 0xd98Be00b5D27fc98112BdE293e487f8D4cA57d07];
+    string[] TOKEN_NAME_ARBITRUM=["Wrapped Comet USDC.e", "Wrapped Comet USDC", "Wrapped Comet WETH", "Wrapped Comet USDT"];
```

Comment:
In the original "wcUSDCev3", "wcUSDCv3" have the same name "cUSDCv3", but for the sake of distinction a distinction was added